src/components/NFTListingTable/index.tsx

```
@@ -275,41 +540,88 @@ const NFTListingTable: React.FC<Props> = ({
-                onClick={() => {
-                  if (enableSelect) {
-                    onSelect(`${listing.mpContractId}-${listing.mpListingId}`);
-                  }
-                }}
-                selected={enableSelect && selected === pk}
-                hover={true}
-                key={index}
-              >
-                {columns.includes("timestamp") ? (
+              <>
+                <NFTListingTableRow listing={listing} />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)